apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -8,38 +8,77 @@
             "Pre-configured addresses of epoch nodes to contact",
             "type"  : "array",
             "items" : {
-                "type"    : "string",
-                "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$",
-                "example" : "http://somehost.somedomain:123/"
+                "type" : "object",
+                "properties" : {
+                    "peer" : {
+                        "type" : "object",
+                        "additionalProperties" : false,
+                        "properties" : {
+                            "host" : {
+                                "type"    : "string",
+                                "pattern" : "^[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*",
+                               "example" : "somehost.somedomain:123"
+                            },
+
+                            "port" : {
+                                "type" : "integer",
+                                "minimum" : 1,
+                                "maximum" : 65535
+                            },
+
+                           "pubkey" : {
+                               "type" : "string",
+                               "description" : "base58c encoded public key"
+                          }
+                        }
+                    }
+                }
             }
         },
         "blocked_peers" : {
             "description" :
             "Pre-configured addresses of epoch nodes NOT to contact",
             "type"  : "array",
             "items" : {
-                "type"    : "string",
-                "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$",
-                "example" : "http://somehost.somedomain:123/"
+                "type" : "object",
+                "properties" : {
+                    "peer" : {
+                        "type" : "object",
+                        "additionalProperties" : false,
+                        "properties" : {
+                            "host" : {
+                                "type"    : "string",
+                                "pattern" : "^[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*",
+                               "example" : "somehost.somedomain:123"
+                            },
+
+                            "port" : {
+                                "type" : "integer",
+                                "minimum" : 1,
+                                "maximum" : 65535
+                            },
+
+                           "pubkey" : {
+                               "type" : "string",
+                               "description" : "base58c encoded public key"
+                          }
+                        }
+                    }
+                }
             }
         },
         "sync" : {
             "type" : "object",
             "additionalProperties" : false,
             "properties" : {
+                "port" : {
```

Comment:
I'd like to see `listen_address` as well (see my other commend)

Reply:
adding to the backlog